bindings/Python/py11File.cpp

```diff
@@ -354,5 +360,31 @@ adios2::Mode File::ToMode(const std::string mode) const
     return modeCpp;
 }
 
+Layout File::ToLayout(std::string order)
```

Comment:
`Layout File::ToLayout( const std::string order) const`

Reply:
It's a static member function, so it can't be const.

 I guess one can argue about whether it should be a const (non-static) member function instead, but since it doesn't use `this`, there's really no point. In fact, it could be made a free function instead, which I thought about, but well, since there aren't any other free functions around there, I figured this keeps it with where it's used.

Reply:
It can be moved to the `adios2::helper` namespace that contains all the common reusable free functions. See my comment above about `static`.

Reply:
I'll move it to `helper` once / if it gains more than one user. It's not generally usable, since it's python-interface specific. But it might get used in the low-level interface eventually,
at which point it makes sense to share it.